cdap-ui/app/cdap/components/PipelineDetails/store/index.js

```diff
@@ -69,6 +69,8 @@ const DEFAULT_PIPELINE_DETAILS = {
   name: '',
   type: '',
   description: '',
+  artifactVersion: '',
+  artifactScope: '',
```

Comment:
wouldn't this be better to just store as the artifact full object? combine type, artifactVersion, and artifactScope?